ledger/participant-integration-api/src/main/scala/platform/store/FlywayMigrations.scala

```diff
@@ -53,8 +53,9 @@ private[daml] class FlywayMigrations(
   def migrate(allowExistingSchema: Boolean = false): Future[Unit] = run { configBase =>
```

Comment:
shouldn't you would need to do something similar for the validate and validateAndWait as well?

In general I am concerned that we do not have test in daml for migrations. We have another task to tackle that aspect. I am wondering whether it would make sense first doing that one, and only add these changes when we have means to test it.